spec/swagger.yaml

```diff
@@ -5569,6 +5598,64 @@ definitions:
         },
         "included": []
       }
+  UpsertPatientRequest:
+    type: object
+    required:
+      - meta
+      - data
+    properties:
+      meta:
+        type: object
+        required:
+          - query
+        properties:
+          query:
+            type: object
+            required:
+              - identifier
+              - groups
+            description: Query identifying group to create patient in, and identifier to match patient
```

Comment:
The `groups` in query seems to be the list of groups for filtering in determining the patient to be update, not to create a new patient in.

Reply:
since it's an upsert, you can either create or update in that group. I can update the documentation